refresh/templates/services/watsonconversation/importDependency.swift

```diff
@@ -0,0 +1 @@
+        .Package(url: "https://github.com/watson-developer-cloud/swift-sdk",    majorVersion: 0),
```

Comment:
As we discussed in scrum, we will address questions with how to represent dependency versions at our next sprint planning meeting